backend/apps/slack/migrations/0014_message.py

```
@@ -0,0 +1,32 @@
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('slack', '0013_alter_conversation_total_members_count_and_more'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Message',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)